desci-server/src/scripts/test-exec.ts

```
@@ -1,17 +1,15 @@
-import { prisma } from '../client.js';
 import { logger as parentLogger } from '../logger.js';
-import { ElasticNodesService } from '../services/ElasticNodesService.js';
-import { ensureUuidEndsWithDot } from '../utils.js';
+import { MergeUserService } from '../services/user/merge.js';
 
 const logger = parentLogger.child({ module: 'SCRIPTS::Testing' });
 
 async function testExec() {
   // debugger;
   logger.info('Testing script run');
+  debugger;
```

Biome (1.9.4)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)